File: 3oy1 Muhriddin Sobirjonov/3masala.c
#include    <stdio.h>
#include    <stdlib.h>

void    ALMASH(int *a, int *b, int *c){
    int son = *c;
    *c = *b;
    *b = *a;
    *a = son;
}

int main(){
    system("cls");
    int a, b, c;
    printf("3 ta son kiriting:  ");
    scanf("%d%d%d", &a, &b, &c);

    printf("son1(A) ==>  %d\n", a);
    printf("son1(B) ==>  %d\n", b);
    printf("son1(C) ==>  %d\n", c);
    puts("\n\n");
    
    ALMASH(&a, &b, &c);
    printf("son1(A) ==>  %d\n", a);
    printf("son1(B) ==>  %d\n", b);
    printf("son1(C) ==>  %d\n", c);   


    return 0;
}

File: 4oy1 Muhriddin Sobirjonov/3masala.py
import  os 
os.system("cls")

a, b, c = map(int, input("3 ta son kiriting  ").split())

if  a >= b and a <= c:
    print("ortacha qiymat:  ", a)
elif a <= b and a >= c:
    print("ortacha qiymat:  ", a)
elif    b >= a and b <= c:
    print("ortacha qiymat:  ", b)
elif    b <= a and b >= c:
    print("ortacha qiymat:  ", b)
elif    c >= a and c <= b:
    print("ortacha qiymat:  ", c)
elif    c <= a and c >=b:
    print("ortacha qiymat:  ", c)
    
    
    